### ISSUE TITLE
ANDIF if being treated as Trigger and not as a Condition

### ISSUE THREAD

**Issue by mcsSolutions** [HomeSeer/Plugin-SDK#338]
Reference Message Board https://forums.homeseer.com/forum/hs4-products/hs4-plugins/lighting-primary-technology-plug-ins-aa/hubitat-elevation-michael-mcsharry-aa/1444958-nuheat-thermostats/page2 post 25, 26

A Device Feature exists with control values of -1, -2, -3 and status values of 0, 1 and 2.  An event is setup where this Feature is the subject of the ANDIF condition.  The options to select the conditional state are the control values (-1, -2, or -3) and they should be the status values (0, 1 or 2).  

In summary, an event that involves a Device Feature should present to the user Control Values for IF and Status Values for ANDIF.

**Comment by spudwebb** [comment]
Why would a trigger use Control values? I think both trigger (IF) and conditions (ANDIF) should use status values.

I saw in the forum thread that this user is using the legacy event page to edit this event, this is likely to be the problem, the legacy page does not handle very well HS4 devices. I just tested with the HS4 event page and it works as expected, but I see the problem in the legacy page.

**Comment by mcsSolutions** [comment]
I agree with you.  It looks as if HS (Legacy) is only providing control values as part of the event setup.  Feel free to respond to user in the thread as to what he should do.
